server/svix-server/Cargo.toml

```
@@ -51,7 +51,7 @@ sea-orm = { version = "0.7.1", features = [ "sqlx-postgres", "runtime-tokio-rust
 sqlx = { version = "0.5", features = [ "runtime-tokio-rustls", "postgres", "migrate" ] }
 http = "0.2"
 time = { version = "0.3.9", features = [ "std" ]}
-redis_cluster_async = "0.7.0"
+redis_cluster_async = { git = "https://github.com/svix/redis-cluster-async" }
```

```suggestion
redis_cluster_async = { git = "https://github.com/svix/redis-cluster-async", rev = "COMMIT_HASH" }
```
Set to a specific commit hash.